docs/assets/center_img.jsx

```diff
@@ -0,0 +1,26 @@
+import React from "react";
+
+/**
+ * Component for centering an image in the docs
+ *
+ * @param {*} props :
+ *  src: image source
+ *  alt: image alternative text
+ *  imgStyle: object, any image styles (props can be passed in as: `imgStyle={{ maxHeight: "600px", border: "solid" }}`)
+ * @returns a component for uniformally centering images in the doc
+ */
+
+export default function CenterImg(props) {
+  let imgStyleString;
+  if (props.imgStyle != undefined && Object.keys(props.imgStyle).length === 0) {
+    imgStyleString = {};
+  } else {
+    imgStyleString = props.imgStyle;
+  }
```

Comment:
A great way of doing [prop delegation](https://courses.joshwcomeau.com/joy-of-react/04-component-design/03-delegating-props) is by using the [spread operator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Spread_syntax)

Reply:
I see! This looks really nice and makes more sense to keep things clean. I tried to modify the current `CenteredImage` component to use this! Thank you!